.github/workflows/ci.yaml

```diff
@@ -13,6 +13,7 @@ jobs:
     env:
       PROTOCOL_DEPLOYER_PRIVATE_KEY: 123456789abcdef123456789abcdef123456789abcdef123456789abcdef1234
       CC_TOKEN_DEPLOYER_PRIVATE_KEY: 123456789abcdef123456789abcdef123456789abcdef123456789abcdef1234
+      CMC_API_KEY: ${{ secrets.CMC_API_KEY_NEW }}
```

Comment:
So we determined that there was some clobbering or mismatch of the key in the secrets object. But can we change the variable to be something like COINMARKETCAP_API_KEY? The names above it are verbose, so it doesn't need to be abbreviated or have NEW in it.

Reply:
I agree. I have renamed the secret variable to `COINMARKETCAP_API_KEY`. PR is now ready for review. @cliffhall

Reply:
Cool, thanks